etrago/tools/calc_results.py

```diff
@@ -22,290 +22,752 @@
 calc_results.py defines methods to calculate results of eTraGo
 """
 import os
-if 'READTHEDOCS' not in os.environ:
-    import time
+
+if "READTHEDOCS" not in os.environ:
     import logging
 
     import pandas as pd
-    import numpy as np
 
     logger = logging.getLogger(__name__)
 
-__copyright__ = ("Flensburg University of Applied Sciences, "
-                 "Europa-Universität Flensburg, "
-                 "Centre for Sustainable Energy Systems, "
-                 "DLR-Institute for Networked Energy Systems")
+__copyright__ = (
+    "Flensburg University of Applied Sciences, "
+    "Europa-Universität Flensburg, "
+    "Centre for Sustainable Energy Systems, "
+    "DLR-Institute for Networked Energy Systems"
+)
 __license__ = "GNU Affero General Public License Version 3 (AGPL-3.0)"
 __author__ = "ulfmueller, s3pp, wolfbunke, mariusves, lukasol"
 
 
 def _calc_storage_expansion(self):
-        """ Function that calulates storage expansion in MW
+    """Function that calulates storage expansion in MW
+
 
+    Returns
+    -------
+    float
+        storage expansion in MW
 
-        Returns
-        -------
-        float
-            storage expansion in MW
+    """
+    return (
+        (
+            self.network.storage_units.p_nom_opt
+            - self.network.storage_units.p_nom_min
+        )[self.network.storage_units.p_nom_extendable]
+        .groupby(self.network.storage_units.carrier)
+        .sum()
+    )
 
-        """
-        return (self.network.storage_units.p_nom_opt -
-                self.network.storage_units.p_nom_min
-                )[self.network.storage_units.p_nom_extendable]\
-                    .groupby(self.network.storage_units.carrier).sum()
 
 def _calc_store_expansion(self):
-        """ Function that calulates store expansion in MW
+    """Function that calulates store expansion in MW
 
-        Returns
-        -------
-        float
-            store expansion in MW
+    Returns
+    -------
+    float
+        store expansion in MW
+
+    """
+    return (self.network.stores.e_nom_opt - self.network.stores.e_nom_min)[
+        self.network.stores.e_nom_extendable
+    ]
 
-        """
-        return (self.network.stores.e_nom_opt -
-                self.network.stores.e_nom_min
-                )[self.network.stores.e_nom_extendable]
 
 def _calc_sectorcoupling_link_expansion(self):
-        """ Function that calulates expansion of sectorcoupling links in MW
+    """Function that calulates expansion of sectorcoupling links in MW
```

Comment:
I actually didn't change this function. There are only changes because of black. 
In addition, those links have no invest costs. It is correct that it would be more consistent to add them here, but the results would be always 0 EUR.